src/Socket.php

```diff
@@ -36,16 +36,12 @@ public function __construct(LoopInterface $loop, $socket, Buffer $buffer = null)
 
     public function getLocalAddress()
     {
-        if ($this->socket !== false) {
-            return stream_socket_get_name($this->socket, false);
-        }
+        return $this->sanitizeAddress(@stream_socket_get_name($this->socket, false));
```

Comment:
There are several occasions where the `stream_socket_get_name()` call may emit a warning and unfortunately this is not reliable across different PHP versions and HHVM. Besides, the `socket` property can accessed from outside of this class so that the `false` check is simply not reliable either.

I've removed the check completely for consistency with the Socket component and simply discard any error messages from this call.